primitives/primproc/batchprimitiveprocessor.cpp

```diff
@@ -2091,15 +2089,19 @@ void BatchPrimitiveProcessor::makeResponse()
         {
             *serialized << (uint8_t) 1;
             *serialized << lbidForCP;
-            if (hasBinaryColumn)
+
+            if (UNLIKELY(hasWideColumnOut))
             {
-                *serialized << (uint8_t) 16; // width of min/max value
-                *serialized << (unsigned __int128) bigMinVal;
-                *serialized << (unsigned __int128) bigMaxVal;
+                // PSA width
+                // Remove the assert for >16 bytes DTs.
+                assert(wideColumnWidthOut == datatypes::MAXDECIMALWIDTH);
+                *serialized << (uint8_t) wideColumnWidthOut;
+                *serialized << min128Val;
+                *serialized << max128Val;
```

Comment:
Again, should be casted to unsigned.

Reply:
I see no reason if ByteStream supports signed.